dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/template/components/SingleSelectComponentFactory.kt

```diff
@@ -21,7 +21,6 @@ class SingleSelectComponentFactory(@Autowired val templateDiagnostic: TemplateDi
         utils: ComponentGenerationUtils,
         componentGroup: ComponentGroupApi,
     ): ComponentBase {
-        templateDiagnostic.optionsNotUsed(row)
```

Comment:
Why was this even there?

Reply:
Same answer as for the Multi-Select component. Copy-paste leftover